packages/session/package.json

```diff
@@ -27,7 +27,7 @@
     "@types/chai": "^4.3.16",
     "@types/cookie": "^0.6.0",
     "@types/cookie-signature": "^1.1.2",
-    "@types/node": "^20.13.0",
+    "@types/node": "^20.14.9",
     "@types/node-fetch": "^2.6.11",
     "@types/on-headers": "^1.0.3",
     "@types/set-cookie-parser": "^2.4.7",
```

Comment:
I was unable to upgrade this dependency: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69734#issuecomment-2197712729

Reply:
PR to fix these types: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69944